Corona-Warn-App/src/test/java/de/rki/coronawarnapp/task/testtasks/timeout/TimeoutTask2.kt

```diff
@@ -0,0 +1,9 @@
+package de.rki.coronawarnapp.task.testtasks.timeout
+
+import javax.inject.Provider
+
+class TimeoutTask2 : BaseTimeoutTask() {
```

Comment:
There is no difference between  TimeoutTask and TimeoutTask2, is there? In that case, he could remove this one !?

Reply:
They are the same, but the TaskController differentiates between two task types (to decide on parallel execution) by comparing the classes, we need a second task class here to test timeouts in parallel execution.